torch/testing/_internal/common_methods_invocations.py

```diff
@@ -18228,12 +18228,25 @@ def error_inputs_mean(op_info, device, **kwargs):
         dtypesIfCUDA=floating_types_and(torch.float16, torch.bfloat16),
         sample_inputs_func=sample_inputs_scatter_reduce,
     ),
-    OpInfo(
-        'scatter_reduce',
-        variant_test_name='amax',
-        dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
-        dtypesIfCUDA=floating_types_and(torch.float16, torch.bfloat16),
-        sample_inputs_func=sample_inputs_scatter_reduce,
+    UnaryUfuncInfo(
+        'special.dawson',
+        decorators=(
+            precisionOverride(
```

Comment:
use toleranceOverride (precisionOverride is deprecated and we should actually remove it sometime)

Those are wide tolerances -- are they reflective of an actual numerical accuracy issue, or is this implementation just divergent from SciPy's but no less accurate?